### PR TITLE
update workflow.GoNamed() doc

### PR DESCRIPTION
**Pull request by mikolajb**

### CODE REVIEW

File: workflow/deterministic_wrappers.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func Go(ctx Context, f func(ctx Context)) {
 
 // GoNamed creates a new coroutine with a given human readable name.
 // It has similar semantic to goroutine in a context of the workflow.
-// Name appears in stack traces that are blocked on this Channel.
+// Name appears in stack traces that are blocked on this coroutine.
```
Comment (mfateev) [inline workflow/deterministic_wrappers.go]: good catch. I would rephrase it to something like: 
Name appears in stack traces that include this coroutine.

Comment (mikolajb) [reply workflow/deterministic_wrappers.go]: Done
